docs/source/process/release_procedure.rst

```diff
@@ -0,0 +1,77 @@
+.. _release_procedure:
+
+Release Procedure
+=================
+
+**Release Manager:** Andre Merzky
+
+Preconditions for release
+-------------------------
+
+1. If release is a milestone release: no open tickets for milestone;
+2. all tests on ``devel`` `pass <https://github.com/radical-cybertools/radical.pilot/actions>`__.
+
+Preparing a regular Release
+---------------------------
+
+1.  Pull ``devel``: ``git checkout devel; git pull``;
+2.  create branch from latest master: e.g. ``git checkout master; git pull; git
+    checkout -b release/0.1.2``;
+3.  update version: ``echo "0.1.2" > VERSION``;
+4.  make modifications to branch: usually by merging devel ``git merge devel``
+    (make sure to pull ``devel`` before);
+5.  update version dependencies to radical stack in setup.py;
+6.  update release notes: ``$EDITOR CHANGES.md``;
+7.  commit and push: ``git commit -a; git push`` (make sure there are no
+    unwanted files in the repo);
+8.  create `pull-request
```

Comment:
We don't do release PRs for releases really.  The current release procedure is captured in this [ticket](https://github.com/radical-cybertools/radical.pilot/issues/2815) - maybe we should copy parts of that information?

Reply:
Thanks @andre-merzky . I copied over some of the text of the indicated ticket but we are still missing the `git-dsyn` script and editing the instructions for an hotfix. Could you do the needful and commit directly to this PR? I do not have the detail to do that myself. Also, you may want to update the page about testing. Also in that case, I do not have enough details to do it myself.